package.json

```diff
@@ -16,55 +16,70 @@
   "typescript": {
     "definition": "dist/index.d.ts"
   },
+  "exports": {
+    ".": "./dist/index.js",
+    "./frontend": "./dist/frontend/index.js",
+    "./package.json": "./package.json"
+  },
+  "typesVersions": {
+    "*": {
+      "frontend": [
+        "dist/frontend/index.d.ts"
+      ]
+    }
+  },
```

Comment:
So this is used to trick ts into adhering to the exports field. Related to your comment on the exports field, it seems that the TS team is not supporting the exports field correctly currently, so this is the workaround so that ts finds the types.